### PR TITLE
Increase MAX_POINTERS_COUNT to 12

### LINKED ISSUE #825: ArrayIndexOutOfBoundsExceptioncom.swmansion.gesturehandler.GestureHandler
I have error. But I not use react-native-gesture-handler in my screens components.
upd: this error appears when I touch screen by palm (multi thouch).

```
java.lang.ArrayIndexOutOfBoundsException: length=11; index=11
    at com.swmansion.gesturehandler.GestureHandler.startTrackingPointer(GestureHandler.java:205)
    at com.swmansion.gesturehandler.GestureHandlerOrchestrator.recordViewHandlersForPointer(GestureHandlerOrchestrator.java:390)
    at com.swmansion.gesturehandler.GestureHandlerOrchestrator.traverseWithPointerEvents(GestureHandlerOrchestrator.java:466)
    at com.swmansion.gesturehandler.GestureHandlerOrchestrator.extractGestureHandlers(GestureHandlerOrchestrator.java:403)
    at com.swmansion.gesturehandler.GestureHandlerOrchestrator.onTouchEvent(GestureHandlerOrchestrator.java:97)
    at com.swmansion.gesturehandler.react.RNGestureHandlerRootHelper.dispatchTouchEvent(RNGestureHandlerRootHelper.java:121)
    at com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView.dispatchTouchEvent(RNGestureHandlerEnabledRootView.java:31)
    at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2583)
    at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2284)
    at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2583)
    at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2284)
    at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2583)
    at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2284)
    at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2583)
    at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2284)
    at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2583)
    at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2284)
    at com.android.internal.policy.PhoneWindow$DecorView.superDispatchTouchEvent(PhoneWindow.java:2403)
    at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1737)
    at android.app.Activity.dispatchTouchEvent(Activity.java:2771)
    at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)
    at com.android.internal.policy.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2364)
    at android.view.View.dispatchPointerEvent(View.java:9520)
    at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4265)
    at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4107)
    at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3653)
    at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3706)
    at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3672)
    at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3798)
    at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3680)
    at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3855)
    at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3653)
    at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3706)
    at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3672)
    at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3680)
    at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3653)
    at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5959)
    at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5933)
    at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5894)
    at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6062)
    at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:185)
    at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
    at android.os.MessageQueue.next(MessageQueue.java:323)
    at android.os.Looper.loop(Looper.java:135)
    at android.app.ActivityThread.main(ActivityThread.java:5422)
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

"react": "16.8.3",
"react-native": "0.59.9",
"react-native-gesture-handler": "^1.3.0",

### PR DESCRIPTION
**Pull request by jakub-gonet**
## Description

Fixes #825.

Apparently there are devices supporting multitouch up to 12 pointers.

## Test plan

Didn't, I only have ten fingers 😢

### CODE REVIEW

**Comment by jkadamczyk** [comment]
@jakub-gonet 
Imagine there is a table with a touchscreen and there are several people interacting with it, now you can have more than 10 fingers 😄

File: android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
   public static final int DIRECTION_UP = 4;
   public static final int DIRECTION_DOWN = 8;
 
-  private static int MAX_POINTERS_COUNT = 11;
```
Comment (jkadamczyk) [inline android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.java:33]: Why it was 11 though? And not 10?

Comment (jakub-gonet) [reply android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.java:33]: I'll @kmagiera, don't really know

**Review by jkadamczyk — APPROVED** [review]

File: android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.java (hunk 1 above)
Comment (jakub-gonet) [reply android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.java:33]: He doesn't know it either, it's how it is 🤷

**Pull request merged**